### ISSUE TITLE
Should apply Accent.Plug.Response without specifying Content-Type: application/json 

### ISSUE THREAD

**Issue by iceniveth** [malomohq/accent#24]
Would it be possible to apply the `Accent.Plug.Response` without having a request with `Content-Type: application/json` header.

For file upload uses cases, the request header has `Content-Type: multipart/form-data`, I should expect the result be transform into json in camel case.

A configuration would look like this:

```elixir
pipeline :uploads do
  plug Accent.Plug.Response,
    default_case: Accent.Case.Camel,
    json_codec: Jason
end

scope "/uploads", StackedWeb do
  pipe_through :uploads

  resources "/files", FileController, only: [:create, :show]
end
```

I've been using [propercase](https://github.com/johnnyji/proper_case), and it does handle this kind of use case.

**Comment by chengchen** [comment]
Yes, `Content-Type` in the request indicates the media type of the request body. And this is even more strange, when we have to put a content-type in a GET operation.
https://tools.ietf.org/html/rfc7231#section-3.1.1.5

**Comment by anthonator** [comment]
Does it make sense to have configurable accepted content types?

**Comment by chengchen** [comment]
@anthonator I am not sure what you meant by `configurable accepted content types`.

From the user point of view, the rules are simple:
* Accent should apply the transformation as long as the response is in json (with `content-type: application/json` in the response headers), otherwise skip the operation.
* If the user prefers `Accent` way of transforming the json keys (using Plug), it means we would like to apply it in a **uniformed** way, as long as the first rule applies.

In other words, the content-type of the request has nothing to do with the decision to transform or not. At least that's what I understood from @iceniveth 's explanation.

**Comment by anthonator** [comment]
Thanks for the explanation.

It looks like transforming the response is ignored if the request `Content-Type` is not `application/json`. This is probably wrong. The request `Content-Type` should not have bearing on the response `Content-Type`.

I'll look into this. Thanks for reporting.

**Comment by anthonator** [comment]
Fixed via #32
